bin/run_server.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2016 The Johns Hopkins University/Applied Physics Laboratory
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import optparse
+import sys
+
+from kmip.services import server
+
+
+def build_argument_parser():
+    parser = optparse.OptionParser(
+        usage="%prog [options]",
+        description="Run KMIP client operation")
+
+    parser.add_option(
+        "-n",
+        "--hostname",
+        action="store",
+        type="str",
+        default=None,
+        dest="hostname",
+        help=(
+            "A string representing either a hostname in Internet domain "
+            "notation or an IPv4 address. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-p",
+        "--port",
+        action="store",
+        type="int",
+        default=None,
+        dest="port",
+        help=(
+            "An integer representing a port number. Recommended to be "
+            "5696 according to the KMIP specification. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-c",
+        "--certificate_path",
+        action="store",
+        type="str",
+        default=None,
+        dest="certificate_path",
+        help=(
+            "A string representing a path to a PEM-encoded server "
+            "certificate file. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-k",
+        "--key_path",
+        action="store",
+        type="str",
+        default=None,
+        dest="key_path",
+        help=(
+            "A string representing a path to a PEM-encoded server "
+            "certificate key file. Defaults to None."
+        ),
+    )
+    parser.add_option(
+        "-a",
+        "--ca_path",
+        action="store",
+        type="str",
+        default=None,
+        dest="ca_path",
+        help=(
+            "A string representing a path to a PEM-encoded certificate "
+            "authority certificate file. Defaults to None."
```

Comment:
Can the file have more than one CA root certificate? My guess is that it is like SSL and can have multiple entries. Might be good to say the format of the file.